cmake/FindLibUV.cmake

```diff
@@ -75,6 +75,7 @@ if(WIN32)
   list(APPEND LIBUV_LIBRARIES ws2_32)
 endif()
 
+find_package(Threads)
```

Comment:
should it be like this one: https://github.com/neovim/neovim/blob/e8fb0728e220bb378a8689415c3915fe5912e987/CMakeLists.txt#L505

    find_package(Threads REQUIRED)

Reply:
Our `Find` files need to be refactored anyway.  This works fine and the main CMakeLists.txt will enforce the "required" aspect of it.  I think this is good enough for now.